### PR TITLE
Use xo code style

### LINKED ISSUE #37: Add a linter and CI
We should add a linter to manage code style. Personally, I’m in favor of [standard](http://standardjs.com), but another style would be fine. We should:

- [ ] add a Travis script that runs tests on all of the `*.user.js` files
- [ ] add a way to install the linter locally and test before commiting

### PR DESCRIPTION
**Pull request by j-f1**
This uses the [xo](https://github.com/sindresorhus/xo) style.

Since lots of stuff was changed, I’m waiting to make sure everyone agrees.

Fixes #37.

### CODE REVIEW

File: gas-mask-se.user.js [hunk 1]
```diff
@@ -14,30 +14,34 @@
 // @version       1.3
 // ==/UserScript==
 
-if(true || location.search.indexOf("smokeypost=true") !== -1){
+if (true || location.search.indexOf("smokeypost=true") !== -1) { // eslint-disable-line no-constant-condition
```
Comment (rschrieken) [inline gas-mask-se.user.js]: Why don't you introduce a NOTFOUND const so yo can do without the eslint supression?

Comment (ArtOfCode-) [inline gas-mask-se.user.js]: @honnza ^

File: autoflagging.user.js [hunk 2]
```diff
@@ -272,38 +225,39 @@
...
         }
       });
 
       if (data.has_more) {
         // There are more items on the next 'page'
         autoflagging.callAPI(urls, ++page);
       }
-    }).fail(function(xhr, textStatus, error) {
-      autoflagging.notify('Failed to load data: ' + xhr.statusText);
+    }).fail(function (xhr, /* textStatus, error */) {
```
Comment (ferrybig) [inline autoflagging.user.js]: Are you sure this is valid javascript? In google chrome I get `Uncaught SyntaxError: Unexpected token )` for `var test = function(a, /* b, c */) {};`

File: autoflagging.css [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+  color: inherit;
+  text-decoration: none !important;
+}
+.ai-flag-count::after {
+  content: " ⚑";
+}
+.ai-flag-count.ai-not-autoflagged::after {
+  content: "⚑";
+}
+;
```
Comment (ArtOfCode-) [inline autoflagging.css]: What are these random semicolons doing here?

**Comment by honnza** [comment]
Since there doesn't seem to be any effort to add the argument this
condition tests, we should remove the condition entirely

File: autoflagging.user.js (hunk 2 above)
Comment (ArtOfCode-) [inline autoflagging.user.js]: It's not - the comma after `xhr` needs to go

**Pull request merged**